using-mobile-extensions/adobe-campaign-standard/adobe-campaign-standard-event-reference.md

```diff
@@ -0,0 +1,178 @@
+
+## Events handled
+
+### Campaign Request Content  <a id="configuration-response-content"></a>
+
+This event is dispatched from Event Hub for the following scenarios:
+1. Event data containing loaded consequences for the latest registered campaign rules needs to be dispatched
+2. When a rule is matched for a Campaign consequence. This event contains the data of either local, alert or full screen message that is triggered to be displayed.
+
+#### Data payload definition  <a id="data-payload-definition-1"></a>
+
+Here are the key-value pairs in this event:
+
+| **Key** | **Value Type** | **Optional** | **Description** |
+| :--- | :--- | :--- | :--- |
+| `triggeredconsequence`| Array |
+| `loadedconsequences` | Array |
+
+#### Event data example
+
+```text
+{ 
+    "loadedconsequences": [
+    {
+        "assetsPath":"assets/path/", 
+        "detail":"details", 
+        "id":"123", 
+        "type":"iam"
+    },
+    {
+        "assetsPath":"assets/path/", 
+        "detail":"details", 
+        "id":"456", 
+        "type":"csp"
+    }]
+}
+```
+
+
+### Configuration Response Content  <a id="configuration-response-content"></a>
+
+The data property in this event is used by each extension to modify its current settings. Each extension is responsible for reading out the part of the data property for which it is concerned.
+
+#### Data payload definition  <a id="data-payload-definition-3"></a>
+
+The Adobe Campaign Standard extension will read the following key from the configuration event:
+
+| **Key** | **Value Type** | **Optional** | **Description** |
+| :--- | :--- | :--- | :--- |
+| `__dev__campaign.server` | String | Yes | This contains the endpoint URL for the development environment Adobe Campaign Standard instance |
+| `__dev__campaign.pkey` | String | Yes | This contains the identifier for a mobile app configured in development environment Adobe Campaign Standard |
+| `__stage__campaign.server` | String | Yes | This contains the endpoint URL for the stage environment Adobe Campaign Standard instance |
+| `__stage__campaign.pkey` | String | Yes | This contains the identifier for a mobile app configured in stage environment Adobe Campaign Standard |
+| `campaign.server` | String | Yes | This contains the endpoint URL for the production environment Adobe Campaign Standard instance |
+| `campaign.pkey` | String | Yes | This contains the identifier for a mobile app configured in production environment Adobe Campaign Standard |
+| `campaign.mcias` | String | Yes | This contains the in-app messaging service URL endpoint |
+| `campaign.timeout` | String | Yes | This contains the amount of time to wait for a response from in-app messaging service |
+| `global.privacy` | Boolean | Yes | This contains the mobile privacy status settings |
+
+{% hint style="info" %}
+If `global.privacy` is changed to **optout**, linkage fields will be reset. All downloaded messages, rules will be erased and no tracking request may leave the device.
```

Comment:
Please reword as follows:

"After `global.privacy` is changed to **optout**, the linkage fields are reset. All downloaded messages and rules are erased, and no tracking request can leave the device."